common/src/main/java/com/wynntils/screens/maps/PoiManagementScreen.java

```diff
@@ -67,79 +111,756 @@ public void onClose() {
+                .customPois
+                .get()
+                .isEmpty()) {
+            FontRenderer.getInstance()
+                    .renderText(
+                            poseStack,
+                            StyledText.fromComponent(
+                                    Component.translatable("screens.wynntils.poiManagementGui.noPois")),
+                            (int) (dividedWidth * 32),
+                            (int) (dividedHeight * 32),
+                            CommonColors.WHITE,
+                            HorizontalAlignment.CENTER,
+                            VerticalAlignment.MIDDLE,
+                            TextShadow.NORMAL);
+
+            return;
+        }
+
+        FontRenderer.getInstance()
+                .renderText(
+                        poseStack,
+                        StyledText.fromComponent(Component.translatable("screens.wynntils.poiManagementGui.search")),
+                        (int) (dividedWidth * 10) + 5,
+                        (int) dividedHeight,
+                        CommonColors.WHITE,
+                        HorizontalAlignment.LEFT,
+                        VerticalAlignment.TOP,
+                        TextShadow.NORMAL);
+
+        if (pois.isEmpty()) {
+            FontRenderer.getInstance()
+                    .renderText(
+                            poseStack,
+                            StyledText.fromComponent(
+                                    Component.translatable("screens.wynntils.poiManagementGui.noFilteredPois")),
+                            (int) (dividedWidth * 32),
+                            (int) (dividedHeight * 32),
+                            CommonColors.WHITE,
+                            HorizontalAlignment.CENTER,
+                            VerticalAlignment.MIDDLE,
+                            TextShadow.NORMAL);
+        } else {
+            RenderUtils.drawRect(
+                    poseStack,
+                    CommonColors.WHITE,
+                    (int) (dividedWidth * 12),
+                    (int) (dividedHeight * HEADER_HEIGHT),
+                    0,
+                    (int) (dividedWidth * 38),
+                    1);
+        }
+    }
+
+    @Override
+    public void renderBackground(GuiGraphics guiGraphics, int mouseX, int mouseY, float partialTick) {
+        super.renderBackground(guiGraphics, mouseX, mouseY, partialTick);
+
+        RenderUtils.drawScalingTexturedRect(
+                guiGraphics.pose(),
+                Texture.WAYPOINT_MANAGER_BACKGROUND.resource(),
+                backgroundX,
+                backgroundY,
+                0,
+                backgroundWidth,
+                backgroundHeight,
+                Texture.WAYPOINT_MANAGER_BACKGROUND.width(),
+                Texture.WAYPOINT_MANAGER_BACKGROUND.height());
+    }
+
+    public void selectPoi(CustomPoi selectedPoi) {
+        boolean add = true;
+        boolean updateState = false;
```

Comment:
`updateState` is not really needed, `add` can hold the logic.. Also, please give it a better name